backend-diagram-converter/core/src/main/java/org/camunda/community/migration/converter/expression/ExpressionTransformer.java

```diff
@@ -75,4 +75,16 @@ private String handleExpression(String expression) {
     }
     return replaced;
   }
+
+  public static boolean hasMethodInvocation(String expression) {
```

Comment:
The `ExpressionTransformationResult` could contain information like this

Reply:
I moved the logic into the `ExpressionTransformationResult`. Looks more like object-oriented style now.